### DISCUSSION TITLE
Someone is imitating you, and I'm not comfortable with it.. https://img.aitoolbox.org/

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [lllyasviel/Fooocus#2428] [General]
Shall we report on this?

**Comment by mashb1t** [comment]
Please elaborate. I don't see any connection to Fooocus.

**Reply by Amit30swgoh** [reply]
I may be wrong it's just look almost identical, and you guys worked so hard to make it

**Reply by mashb1t** [reply]
Fooocus is using Gradio, a python frontend framework, in default styling. Every app based on Gradio without themeing or custom styles will look the same, but obv. does not have the same functions.
Thank you anyways for your commitment.

**Reply by Amit30swgoh** [reply]
I'll close it
